config.go

```diff
@@ -38,6 +38,9 @@ type MinValuableConfig struct {
 	HubPassword string   `toml:"hub_password" commented:"true"`
 }
 
+type ConfigsLogs struct {
```

Comment:
Not sure why both words in the type name are plural. I think it's better to name like `LogFilesConfig`

Reply:
done